client_core/src/com/mygdx/game/client_core/network/GameConnectService.java

```diff
@@ -24,7 +24,7 @@ public GameConnectService(
 
   public void connect() {
     log.info("connect request sent");
-    webSocket.send("connect:" + playerInfo.getUserName() + ":" + playerInfo.getToken());
+    webSocket.send("connect:" + playerInfo.getUserName() + ":" + playerInfo.getToken() + ":default");
```

Comment:
co to jest to default

Reply:
to jest id pokoju, którego się domagają klienci przez zhardkodowanie - piszę teraz drugą część tego issue gdzie można pisać do jakiego pokoju chce się łączyć